app/controllers/users.js

```diff
@@ -0,0 +1,19 @@
+const bcryptjs = require('bcryptjs');
+const userService = require('../services/users');
+const varParse = require('../serializers/varParse');
+const authConfig = require('../../config/auth');
+const logger = require('../logger');
+
+exports.signUp = async (req, res, next) => {
+  try {
+    const password = bcryptjs.hashSync(req.body.password, Number(authConfig.rounds));
```

Comment:
Controllers files should be as clean as posible.
Read [this guide](https://github.com/Wolox/tech-guides/blob/master/nodejs/docs/node-standard-and-best-practices.md#24--helpers) and think if the encrypt method should be in a helper path.

Also keep in mind that the encryption method could be use for any other function or route so you need to put it in a global file.